umbra-js/src/classes/Umbra.ts

```diff
@@ -43,6 +48,20 @@ const subgraphs = {
   42161: 'https://api.thegraph.com/subgraphs/name/scopelift/umbraarbitrumone',
 };
 
+interface SendBatch {
+  token: string;
+  amount: BigNumberish;
+  address: string;
+}
+
+interface SendData {
+  receiver: string;
+  tokenAddr: string;
+  amount: BigNumberish;
+  pkx: string;
+  ciphertext: string;
+}
```

Comment:
Let's define these in `types.ts` instead (seems `SendBatch` is already there)

Reply:
Moved them in 4618fab3d65ea08e70613c8af301ae13e03e0b66